### DISCUSSION TITLE
Limited access of a user

### DISCUSSION THREAD

**Discussion by wolflu05** [inventree/InvenTree#4104] [Q&A] [answered]
Hello, is it possible to limit the access of a user to a particular subtree of locations and part categories, so that this specific user only can see everything in the food category and in the kitchen location? And how can I limit the access to manufacturers and suppliers?

**Comment by matmair** [comment, marked as answer]
@wolflu05 this is currently not possible.
You can restrict what principle objects (all orders, all parts, ...) users have access to / what kind of access (only read, read + write, delete) they have.

There is an FR for something similar: https://github.com/inventree/InvenTree/issues/2323 - fair warning: this is a pretty involved thing and we have to think about the migration journey.

**Reply by wolflu05** [reply]
Ok. Thank you. The easiest solution than may be to set up multiple instances. I know, these permission things are hard problems that need to be thought of properly so that they are no holes in it.
